lib/experimental/class-wp-theme-json-gutenberg.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * WP_Theme_JSON_6_1 class
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Class that encapsulates the processing of structures that adhere to the theme.json spec.
+ *
+ * This class is for internal core usage and is not supposed to be used by extenders (plugins and/or themes).
+ * This is a low-level API that may need to do breaking changes. Please,
+ * use get_global_settings, get_global_styles, and get_global_stylesheet instead.
+ *
+ * @access private
+ */
+class WP_Theme_JSON_Gutenberg extends WP_Theme_JSON_6_1 {
```

Comment:
This enables us to always reference `WP_Theme_JSON_Gutenberg` in experimental libs, rather than targetting a specific version.